web/book/src/project/integrations/README.md

```diff
@@ -2,12 +2,13 @@
 
 PRQL is building integrations with lots of external tools, including:
 
-- [Jupyter](./jupyter.md)
+- [`prqlc` CLI](./prqlc-cli.md) - Rust compiler for the command line
 - [ClickHouse](./clickhouse.md)
+- [Jupyter](./jupyter.md)
 - [DuckDB](./duckdb.md)
+- [qStudio](./qstudio.md)
 - [Prefect](./prefect.md)
 - [VS Code](./vscode.md)
 - [Rill](./rill.md)
 - [PostgreSQL](./postgresql.md)
-
-We also have a CLI, [`prqlc`](./prqlc-cli.md)
+- [Syntax highlighting](./syntax-highlighting.md) for many popular tools.
```

Comment:
Thanks for syncing these!

(kinda agree re the CLI, though only one so it doesn't really make a category...)